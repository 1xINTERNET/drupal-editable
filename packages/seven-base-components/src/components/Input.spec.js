/* eslint-env jest */

import React from "react";
import { shallow } from "enzyme";

import { Input } from ".";

describe("Input", () => {
  let component;

  beforeEach(() => {
    component = shallow(
      <Input onChange={() => {}} value="testValue" label="testLabel" />
    );
  });

  it("should render correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
