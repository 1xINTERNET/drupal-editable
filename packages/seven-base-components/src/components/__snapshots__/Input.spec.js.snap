// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render correctly 1`] = `
<div
  className="form-item editable--form-item editable--form-item--input"
>
  <label
    className="editable--label"
    htmlFor="input_1"
  >
    testLabel
  </label>
  <div
    className="editable--input-wrapper css-1psltl0"
  >
    <input
      className="form-text editable--form-text editable--form-text--input"
      data-prop-path={null}
      disabled={false}
      id="input_1"
      onChange={[Function]}
      placeholder=""
      type="text"
      value="testValue"
    />
  </div>
</div>
`;
